pkg/spec/storage.go

```diff
@@ -663,10 +663,15 @@ func (config *CreateConfig) getVolumeMounts() (map[string]spec.Mount, map[string
 
 		if strings.HasPrefix(src, "/") || strings.HasPrefix(src, ".") {
 			// This is not a named volume
+			// convert src to an absolute path
+			source, err := filepath.Abs(src)
```

Comment:
This is a bad place to handle things since it only hits the `--volume` flag and not `--mount`. Better place would be the loop at https://github.com/containers/libpod/blob/943c8daf6db1ec9995f0e7a8f1392c2d83dc4376/pkg/spec/storage.go#L203-L214 - after we check for tmpfs, check if it's a bind mount, and if it is do a filepath.Abs() on the source path.